### ISSUE TITLE
Update documentation to note that authoredAt timestamp is in milliseconds

### ISSUE THREAD

**Issue by rdawemsys** [comeara/pillar#24]
The example migrations in README.md have authoredAt timestamps in seconds, but they need to be in milliseconds.

If you don't put them in milliseconds, all the authored_at timestamps in Cassandra will be incorrect (in 1970). See issue #21 for more details.

**Comment by shayangz** [comment]
+1 for this. would save a ton of time!

**Comment by jhungerford** [comment]
This bit me too - I updated the seconds in the doc to be milliseconds in the attached PR.

**Comment by ogeagla** [comment]
Big :+1:

**Comment by PeterLappo** [comment]
I added an authordAtDate to my fork with human readable dates, its available here.
https://github.com/smr-co-uk/pillar/blob/patch/README.md 
Its version 2.0.2 in a branch called patch not master
It has most of the patches up to end of Feb 2015 and supports Scala 2.11 by default

Also useful to know you can use you can batch commands in single migration with this syntax in either up: or down:. Believe that works with the official version, but haven't tried it.
```xml
BEGIN BATCH  
<cql script with semicolon at the end>  
<cql script with semicolon at the end>  
<cql script with semicolon at the end>  
APPLY BATCH
```

**Comment by comeara** [comment]
This was addressed in v2.1.
